src/code/z_sram_NES.c

```diff
@@ -1,67 +1,1639 @@
+    gSaveContext.save.lotteryCodes[1][2] = Rand_S16Offset(0, 10);
+    gSaveContext.save.lotteryCodes[2][0] = Rand_S16Offset(0, 10);
+    gSaveContext.save.lotteryCodes[2][1] = Rand_S16Offset(0, 10);
+    gSaveContext.save.lotteryCodes[2][2] = Rand_S16Offset(0, 10);
+
+    // Needed to match...
+    for (i = 0; i < 3; i++) {
+        for (j = 0; j < 3; j++) {}
+    }
+
+    i = 0;
+    sp2A = Rand_S16Offset(0, 16) & 3;
+    k = 6;
+    while (i != k) {
+        randSpiderHouse = Rand_S16Offset(0, 16) & 3;
+        if (sp2A != randSpiderHouse) {
+            gSaveContext.save.spiderHouseMaskOrder[i] = randSpiderHouse;
+            i++;
+            sp2A = randSpiderHouse;
+        }
+    }
+
+    do {
+        randBombers = Rand_S16Offset(0, 6);
+    } while (randBombers <= 0 || randBombers >= 6);
+
+    gSaveContext.save.bomberCode[0] = randBombers;
+
+    i = 1;
+    while (i != 5) {
+        k = false;
+
+        do {
+            randBombers = Rand_S16Offset(0, 6);
+        } while (randBombers <= 0 || randBombers >= 6);
+
+        sp2A = 0;
+        do {
+            if (randBombers == gSaveContext.save.bomberCode[sp2A]) {
+                k = true;
+            }
+            sp2A++;
+        } while (sp2A < i);
+
+        if (k == false) {
+            gSaveContext.save.bomberCode[i] = randBombers;
+            i++;
+        }
+    }
+}
+
+SavePlayerData sSaveDefaultPlayerData = {
+    { '\0', '\0', '\0', '\0', '\0', '\0' },             // newf
+    0,                                                  // deaths
+    { 0x3E, 0x3E, 0x3E, 0x3E, 0x3E, 0x3E, 0x3E, 0x3E }, // playerName "        "
+    0x0030,                                             // healthCapacity
+    0x0030,                                             // health
+    0x00,                                               // magicLevel
+    0x30,                                               // magic
+    0x0000,                                             // rupees
+    0x0000,                                             // swordHealth
+    0x0000,                                             // tatlTimer
+    0x00,                                               // magicAcquired
+    0x00,                                               // doubleMagic
+    0x00,                                               // doubleDefense
+    0x00,                                               // unk_1F
+    0xFF,                                               // unk_20
+    0x0000,                                             // owlActivationFlags
+    0xFF,                                               // unk_24
+    SCENE_SPOT00,                                       // savedSceneNum
+};
+
+ItemEquips sSaveDefaultItemEquips = {
+    {
+        { ITEM_SWORD_KOKIRI, ITEM_NONE, ITEM_NONE, ITEM_NONE },
+        { ITEM_SWORD_KOKIRI, ITEM_NONE, ITEM_NONE, ITEM_NONE },
+        { ITEM_SWORD_KOKIRI, ITEM_NONE, ITEM_NONE, ITEM_NONE },
+        { 0xFD, ITEM_NONE, ITEM_NONE, ITEM_NONE },
```

Comment:
Should an ITEM_UNK_FD be added to the item's enum?

Reply:
Ohh yeah, I forgot to add it, whoops